lib/facter/pe_status_check.rb

```diff
@@ -219,6 +219,19 @@
     end
   end
 
+  chunk(:S0029) do
+    next unless PEStatusCheck.replica? || PEStatusCheck.postgres? || PEStatusCheck.primary?
+    # check if concurrnet connections to Postgres approaching 90% defined
+
+    maximum = PEStatusCheck.max_connections.to_i
+    current = PEStatusCheck.cur_connections.to_i
```

Comment:
```suggestion
    maximum = PEStatusCheck.max_connections.to_f
    current = PEStatusCheck.cur_connections.to_f
```

change these to floats to allow the math to work